ibis/backends/dask/execution/selection.py

```diff
@@ -0,0 +1,201 @@
+"""Dispatching code for Selection operations.
+"""
+
+from __future__ import absolute_import
+
+import functools
+import operator
+from typing import Optional
+
+import dask.dataframe as dd
+import numpy as np
+import pandas
+from toolz import concatv
+
+import ibis.expr.operations as ops
+import ibis.expr.types as ir
+from ibis.backends.pandas.execution.selection import (
+    _compute_predicates,
+    compute_projection,
+    compute_projection_table_expr,
+    execute,
+    execute_node,
+    map_new_column_names_to_data,
+    remap_overlapping_column_names,
+)
+from ibis.expr.scope import Scope
+from ibis.expr.typing import TimeContext
+
+from ..execution import constants
+
+
+@compute_projection.register(ir.ScalarExpr, ops.Selection, dd.DataFrame)
+def compute_projection_scalar_expr(
+    expr,
+    parent,
+    data,
+    scope: Scope = None,
+    timecontext: Optional[TimeContext] = None,
+    **kwargs,
+):
+    name = expr._name
+    assert name is not None, 'Scalar selection name is None'
+
+    op = expr.op()
+    parent_table_op = parent.table.op()
+
+    data_columns = frozenset(data.columns)
+
+    scope = scope.merge_scopes(
+        Scope(
+            {
+                t: map_new_column_names_to_data(
+                    remap_overlapping_column_names(
+                        parent_table_op, t, data_columns
+                    ),
+                    data,
+                )
+            },
+            timecontext,
+        )
+        for t in op.root_tables()
+    )
+    scalar = execute(expr, scope=scope, **kwargs)
+    result = pandas.Series([scalar], name=name).repeat(len(data.index))
+    result.index = data.index
+    return dd.from_pandas(result, npartitions=1)
```

Comment:
Why returning a single partition Series here?

Reply:
Fair point - that should be chunked.